app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        cursor_with_actors = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in cursor_with_actors]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f"VALUES (?, ?)",
+            (first_name, last_name,),
+        )
+        self._connection.commit()
+
+    def update(self, id_to_update: int,
+               first_name: str,
+               last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET first_name = ?, last_name = ? "

```

Comment:
I'm not sure if SQL style guides should apply here, but SET clause is usually placed at new line.

Reply:
yes, that is the best SQL practice. The same as using commands in the upper register! Good comment!